packages/n/node-addon-api/xmake.lua

```diff
@@ -0,0 +1,49 @@
+package("node-addon-api")
+
+    set_kind("library", {headeronly = true})
+    set_homepage("https://github.com/nodejs/node-addon-api")
+    set_description("Module for using Node-API from C++")
+    set_license("MIT")
+
+    add_configs("errors", {
+        description = "Choose between 'except', 'noexcept' and 'maybe' to specify how errors should be handled.\n" ..
+            "See https://github.com/nodejs/node-addon-api/blob/main/doc/error_handling.md for more details.",
+        default = "except",
+        type = "string"
+    })
+    add_configs("disable_deprecated", {
+        description = "Control the availability of the deprecated APIs.",
+        default = true,
+        type = "boolean"
+    })
+
+    set_urls("https://github.com/nodejs/node-addon-api/archive/refs/tags/$(version).tar.gz",
+        "https://github.com/nodejs/node-addon-api.git")
+    add_versions("v8.0.0", "42424c5206b9d67b41af4fcff5d6e3cb22074168035a03b8467852938a281d47")
+
+    add_deps("node", {kind = 'binary', system = true})
```

Comment:
no this package in xmake-repo now.

Reply:
I couldn't find it, removing `system = true` gives me this error:
```
error: package(node) not found!
```

Reply:
so you should add node as package first. and add `on_fetch` to find system node in node package at least.